Dockerfile

```
@@ -1,13 +1,20 @@
-FROM registry.access.redhat.com/ubi9/ubi-minimal:9.5-1730489338
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.5-1742914212
 SHELL ["/bin/bash", "-c"]
 USER root
```

Hadolint (2.12.0)

[warning] Last USER should not be root

(DL3002)